### PR TITLE
Uses createCell method from XlsxUtil

### PR DESCRIPTION
**Pull request by joaofreitas-qubit**
Relates with: #qubIT-Omnis-5961

### CODE REVIEW

**Comment by joaofreitas-qubit** [comment]
Can only be merged after xlsutil goes to core-api

**Review by pabrantes — APPROVED** [review]

**Comment by pabrantes** [comment]
@joaofreitas-qubit looks good to me, but let's run this over @anilmamede as well.

File: src/main/java/org/fenixedu/academictreasury/util/ExcelUtils.java [hunk 1]
```diff
@@ -41,118 +41,88 @@
...
-        return result;
+
+        public List<ExcelSheet> getResult() {
+            return this.result;
+        }
+    }
+	
+    public static List<List<String>> readExcel(final InputStream stream, int maxCols) throws IOException {
+        TreasuryDefaultExcelSheetProcessor sheetProcessor = new TreasuryDefaultExcelSheetProcessor(maxCols);
+        ExcelUtil.importExcel(XlsType.XLSX, stream, sheetProcessor, maxCols);
```
Comment (joaofreitas-qubit) [inline src/main/java/org/fenixedu/academictreasury/util/ExcelUtils.java:136]: @anilmamede Should we support xls files too, or xlsx is enough?

Comment (anilmamede) [reply src/main/java/org/fenixedu/academictreasury/util/ExcelUtils.java:136]: The xls is very old. XLSX should be enough

**Review by anilmamede — APPROVED** [review]
I tested the report extraction and the Excel reader and it worked. Looks good for me

**Pull request merged**